File: src/store/store.js
import {createStore, applyMiddleware, compose} from 'redux';
import thunk from 'redux-thunk';
import rootReducer from './reducer';

const initialState = {};
const middleWare = [thunk];
const enhancers = [];

const store = createStore(
  rootReducer,
  initialState,
  compose(
    applyMiddleware(...middleWare),
    ...enhancers,
  ),
);

export default store;

File: lib/store/store.dart
import 'package:redux/redux.dart';
import 'reducer/index.dart';
import './modal/User.dart';

class MainState {
  User user;
  MainState({this.user});
  MainState.init() {
    this.user = User.init();
  }
}

var store = Store<MainState>(appReducer, initialState: MainState.init());
